README.md

```diff
@@ -37,9 +39,13 @@ This file contains paths to find supporting files, meta data, and dummy data to
 name: Figure Image
 description: A `figure` element with `figcaption` and an included `img` element from /base/img/img.html.
 twig: ./figure-image.twig
-sass: ./sass/_figure-image.scss
+sass: ./_figure-image.scss
 script: ./figure-image.js
 category: components
+subcategory: media
+atomic:
```

Comment:
can we keep using `category(required)` and `subcategory(optional)` for patternlab?   it's not necessarily to use atomic terminology or approach in patternlab, we can discuss categories how we are going to group patterns. 
In this case `category` will be equal to `patternType`, not sure about `subcategory` usage in Patternlab, because PL groups patterns by template names inside category dropdown.  Do we have any idea or case for subcategory usage?

Just bringing additional atomic settings may bring some confusion what to use.